frontend/src/app/reducers/browser.js

```diff
@@ -5,9 +5,11 @@ import { isMinFirefoxVersion } from '../lib/utils';
 const userAgent = navigator.userAgent.toLowerCase();
 const isUserAgentFirefox = userAgent.indexOf('firefox') > -1;
 const isUserAgentMinFirefox = isMinFirefoxVersion(isUserAgentFirefox, userAgent, config.minFirefoxVersion);
+const locale = (navigator.language || '').split('-')[0];
```

Comment:
In a meeting, @johngruen said we just wanted to check against the first-choice language.

`navigator.language` [is the first item in `navigator.languages`](https://github.com/mozilla/testpilot/pull/1688).